gatsby-config.js

```diff
@@ -213,5 +213,14 @@ module.exports = {
                 offset: -100,
             },
         },
+        {
+            resolve: 'gatsby-plugin-webpack-bundle-analyser-v2',
+            options: {
+                production: true,
+                disable: !process.env.ANALYZE_BUNDLE_SIZE,
```

Comment:
when will be the case where we need to use this env var?

Reply:
When we run `npm run analyze` we set the `ANALYZE_BUNDLE_SIZE` to `true`